src/scaledmap.jl

```diff
@@ -15,8 +15,8 @@ ScaledMap(λ::S, lmap::A) where {S<:RealOrComplex,A<:LinearMap} =
 Base.size(A::ScaledMap) = size(A.lmap)
 Base.parent(A::ScaledMap) = (A.λ, A.lmap)
 Base.isreal(A::ScaledMap) = isreal(A.λ) && isreal(A.lmap)
-LinearAlgebra.issymmetric(A::ScaledMap) = issymmetric(A.lmap)
-LinearAlgebra.ishermitian(A::ScaledMap) = ishermitian(A.lmap)
+LinearAlgebra.issymmetric(A::ScaledMap) = isreal(A.λ) && issymmetric(A.lmap)
+LinearAlgebra.ishermitian(A::ScaledMap) = isreal(A.λ) && ishermitian(A.lmap)
```

Comment:
Why is that? `A.λ` is just a constant factor. If it's complex,  it shouldn't contradict symmetry?

Reply:
sqrt(-1) * I is not Hermitian.  the diagonal elements must be real.  i agree with jutho here.

Reply:
Right, it's probably not needed for `issymmetric` if you are happy with complex symmetric (which we are). For `ishermitian`, it is necessary though.

Reply:
This doesn't seem to be outdated, yet.